### ISSUE TITLE
Whdload Games run slowly after updating Retropie to 4.4

### ISSUE THREAD

**Issue by marus70** [midwan/amiberry#378]
### Expected behavior

Games installed following the thread below run properly in my PI 3:
https://retropie.org.uk/forum/topic/12656/amiga-amiberry-whdload-holy-grail-amiberry-s-official-thread/370

### Actual behavior

After updating Retropie to 4.4 every game run slowly. 
I've also updated Amiberry from source without resolving the issue.

### Steps to reproduce the behavior

Install Whdload games following this thread:
https://retropie.org.uk/forum/topic/12656/amiga-amiberry-whdload-holy-grail-amiberry-s-official-thread/370
Update Retropie to 4.4

**Comment by midwan** [comment]
@marus70 
Did you also update the emulator along with RetroPie?
Did the same version of the emulator work normally under 4.3, but slower under 4.4 for example?

What versions of the emulator have you tried, exactly? You mentioned the latest from source, but which one did you try before that?

Have you tried the latest available binary release (v2.19 currently)?

**Comment by marus70** [comment]
> @marus70
> Did you also update the emulator along with RetroPie?

Yes I did, a few days ago, once realized the speed issue I updated Amiberry from binary (with no success) then from source (with no success too).

> Did the same version of the emulator work normally under 4.3, but slower under 4.4 for example?
> 

Exactly.

> What versions of the emulator have you tried, exactly? You mentioned the latest from source, but which one did you try before that?
> 

I had 2.14 on Retropie 4.3 and it worked like a charm until update to Retropie 4.4.

> Have you tried the latest available binary release (v2.19 currently)?

Yes I did (with no success).

**Comment by CypherXG** [comment]
2.14 is really an old Version. Try newest.

**Comment by midwan** [comment]
@CypherXG I think he mentioned that he tried v2.19 and the current `master` as well?

@marus70 
Could you please give me some examples of what worked slower, e.g. specific games that behaved differently?

**Comment by marus70** [comment]
I forgot to mention that I have updated Retropie to 4.4 without underlying OS packages (i'm on Jessie, non Stretch). Maybe this helps to find the issue.

**Comment by marus70** [comment]
Horace, I followerd the thread below until about 8 Months ago when everything was ok and my Aminerry plus whdload Were Perfect:
https://retropie.org.uk/forum/topic/12656/amiga-amiberry-whdload-holy-grail-amiberry-s-official-thread
The thread you linked is older, I never seen it before.
Thanks for your support.

**Comment by HoraceAndTheSpider** [comment]
@marus70 I didn’t link a thread?

I was referring to the same one you posted- which dates back several years.

You need to explain what method you are using.

**Comment by HoraceAndTheSpider** [comment]
I think we should close this issue - it refers to following a guide on an external site.

The “supported” route for running WHDLoad games is via the wiki page here. Anything else will be near impossible to diagnose and is likely caused by Config settings.

**Comment by marus70** [comment]
> I think we should close this issue - it refers to following a guide on an external site.
> 
> The “supported” route for running WHDLoad games is via the wiki page here. Anything else will be near impossible to diagnose and is likely caused by Config settings.

Ok: any suggestion to keep part of the work I had done (about settings, configuration, etc) on a new installation?
